### PR TITLE
feat: use BOAT plugin with the latest version to generate code and cl…

### PR DESCRIPTION
**Pull request by torrespro**
…ients

feat: use Java 11
feat: package the OpenAPI specs in a zip not a jar (no code inside anymore)

### CODE REVIEW

File: service-sdk/12.0.0/add-persistence-to-service/example-service/pom.xml [hunk 1]
```diff
@@ -20,7 +20,7 @@
     <name>Backbase :: Digital Banking Services :: example-service</name>
 
     <properties>
-        <java.version>1.8</java.version>
+        <java.version>11</java.version>
```
Comment (christopherm-backbase) [inline service-sdk/12.0.0/add-persistence-to-service/example-service/pom.xml:23]: Do we actually need to specify java version?

File: service-sdk/12.0.0/migrate-to-open-api/messaging-api/pom.xml [hunk 2]
```diff
@@ -11,12 +11,12 @@
...
     <groupId>com.backbase.message</groupId>
     <artifactId>messaging-api</artifactId>
-    <version>1.0.0-SNAPSHOT</version>
+    <version>1.0.0</version>
     <name>Backbase :: messaging-api</name>
 
     <properties>
         <!-- plugin versions -->
-        <boat-maven-plugin.version>0.13.0</boat-maven-plugin.version>
+        <boat-maven-plugin.version>0.14.0</boat-maven-plugin.version>
```
Comment (SteveWyatt74) [inline service-sdk/12.0.0/migrate-to-open-api/messaging-api/pom.xml]: This POM illustrates what OpenAPI archetype 12.1.0 actually generates so should probably stay as 0.13.0 here.

File: service-sdk/12.0.0/migrate-to-open-api/messaging-api/pom.xml [hunk 3]
```diff
@@ -91,6 +91,23 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```
Comment (SteveWyatt74) [inline service-sdk/12.0.0/migrate-to-open-api/messaging-api/pom.xml:95]: 12.1.0 archetype doesn't generate this by default. Should we be asking users to add this plugin?

File: service-sdk/12.0.0/add-persistence-to-service/example-service/pom.xml (hunk 1 above)
Comment (torrespro) [reply service-sdk/12.0.0/add-persistence-to-service/example-service/pom.xml:23]: No, it's part of the SSDK, but not sure if we had it here on purpose to make more explicit so I just updated it

File: service-sdk/12.0.0/migrate-to-open-api/messaging-service/pom.xml [hunk 4]
```diff
@@ -20,19 +21,11 @@
...
     <properties>
-        <java.version>1.8</java.version>
+        <java.version>11</java.version>
+        <boat-maven-plugin.version>0.14.0</boat-maven-plugin.version>
     </properties>
 
     <dependencies>
-
-        <!-- Added spec dependency -->
-        <!-- tag::spec-dependency[] -->
```
Comment (SteveWyatt74) [inline service-sdk/12.0.0/migrate-to-open-api/messaging-service/pom.xml:29]: Removing tag causes asciidoc build to break, can't merge until I've taken this out of asciidoc

**Pull request merged**